### PR TITLE
Fix related() with nullable foreignkeys on django 1.9.

### PR DESCRIPTION
**Pull request by rygwdn**
Ensures that related models are saved before adding, using the same logic as is used for manytomany.

### CODE REVIEW

File: model_mommy/mommy.py [hunk 1]
```diff
@@ -369,6 +369,9 @@ def instance(self, attrs, _commit):
 
     def _handle_one_to_many(self, instance, attrs):
         for k, v in attrs.items():
+            for value in v:
```
Comment (rygwdn) [inline model_mommy/mommy.py]: This is only a problem on Django 1.9, so it could be moved inside the `if` block below, but I put it here to be consistent.

**Comment by tdyer-aio** [comment]
This fixes an issue I am seeing. Is there any timeline for this to get merged?

**Comment by mdentremont** [comment]
+1

**Comment by berinhard** [comment]
@vandersonmota I am a :+1: too. The checking for django version inside the function scope annoys me, but since it was already there, we can mark it as a refactoring to be made...

**Comment by rygwdn** [comment]
@vandersonmota or @berinhard: any idea when there could be a new release with this change?

**Comment by vandersonmota** [comment]
Next month, probably. We still want to make some code cleanup, new fields support, etc.

**Pull request merged**